website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -102,15 +102,10 @@ Where parameters can be:
 - `.withKeyPairAuth(username, privateKey)`
```

Comment:
Ok, I just checked and there is also nothing like this in current version @purbanow

Reply:
And ass well:
`withOAuth`
`withUsernamePasswordAuth`

Reply:
Correct, it's in the newest version

Reply:
removed all 3